ledfx/audio_device_monitor.py

```
@@ -0,0 +1,380 @@
+                    self._stop_event.wait()
+
+                finally:
+                    try:
+                        if hasattr(self, "_device_enumerator") and hasattr(
+                            self, "_notification_client"
+                        ):
+                            self._device_enumerator.UnregisterEndpointNotificationCallback(
+                                self._notification_client
+                            )
+                    except Exception:
+                        pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)